examples/gatsby/src/components/CodeBlock.tsx

```diff
@@ -0,0 +1,43 @@
+import React from 'react';
+import { LightAsync as SyntaxHighlighter } from 'react-syntax-highlighter';
+import javascript from 'react-syntax-highlighter/dist/cjs/languages/hljs/javascript';
+import githubGist from 'react-syntax-highlighter/dist/cjs/styles/hljs/github-gist';
+import { Builder } from '@builder.io/react';
+
+SyntaxHighlighter.registerLanguage('javascript', javascript);
+
+// Example of a custom component you can use in the Builder visual editor
+// See here https://github.com/BuilderIO/builder/tree/master/examples/react-design-system
+// for many more custom component examples
+function CodeBlockComponent({ language, code }: any /* TODO: types */) {
```

Comment:
so this code block component I think is not the best example for us to use, it's pretty particular to our use case and i think all the stuff about "code" and enter code can cause confusion

something that will be more relatable like a simple button or hero, e.g. just strait copy and pasted from our design system example, will be a much more relatable and understandable example for people